astarte/device/device.py

```diff
@@ -767,3 +801,57 @@ def __send_empty_cache(self) -> None:
             retain=False,
             qos=2,
         )
+
+    def __send_set_device_properties(self) -> None:
```

Comment:
What do you think to rename this function ?
- `send_device_owned_properties` ( sdk-rust)
- `send_device_properties`
- ...